source/index.html.md

```diff
@@ -5294,6 +5298,410 @@ Payment Requests where you're the creditor and collecting funds from the debtor.
 |---|---|---|---|
 |200|[OK](https://tools.ietf.org/html/rfc7231#section-6.3.1)|OK|[ListOutgoingPaymentRequestsResponse](#schemalistoutgoingpaymentrequestsresponse)|
 
+## List Payment Requests: Collections
+
+<a id="opIdListOutgoingPaymentRequestCollections"></a>
+
+> Code samples
+
+```shell
+curl --request GET \
+  --url https://api.sandbox.split.cash/payment_requests/collections \
+  --header 'accept: application/json' \
+  --header 'authorization: Bearer {access-token}'
+```
+
+```ruby
+require 'uri'
+require 'net/http'
+
+url = URI("https://api.sandbox.split.cash/payment_requests/collections")
+
+http = Net::HTTP.new(url.host, url.port)
+http.use_ssl = true
+http.verify_mode = OpenSSL::SSL::VERIFY_NONE
```

Comment:
Are we suggesting _not_ to verify the SSL connection? 😕 

```suggestion
http.verify_mode = OpenSSL::SSL::VERIFY_PEER
```

Reply:
Ahhhh this is autogenerated so I do not know anything about this.